packages/scripts/src/node-environments-manager.ts

```diff
@@ -17,15 +17,29 @@ import {
     isEnvironmentStartMessage,
     StartEnvironmentOptions,
     TopLevelConfigProvider,
+    ICommunicationMessage,
 } from './types';
 import type { OverrideConfig } from './config-middleware';
 import { getEnvironmntsForFeature } from './utils/environments';
+import { IPCHost } from '@wixc3/engine-core-node';
 
-type RunningEnvironments = Record<string, number>;
+type RunningEnvironment = {
```

Comment:
interface (which, unlike type, can be extended).

and if it's in the public api, directly or indirectly, export it.